Orange/widgets/data/tests/test_owconcatenate.py

```diff
@@ -17,6 +18,10 @@
 
 class TestOWConcatenate(WidgetTest):
 
+    class DummyTable(Table):
```

Comment:
Is this supposed to be here?
If yes, I am very curious as to what it does.

Reply:
It is used to test mismatching inputs, in line 133.  Both Corpus and DummyTable extend Table class, so we are testing the same thing, without the assumption of text addon being present.

Reply:
I am blind apparently. I did not see L133. Cool stuff! 😎